packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -53,37 +57,40 @@ export const MenuItem: React.FC<MenuItemProps> = React.memo(
                 }
               : undefined
           }
-          rel={target === '_blank' ? 'noopener noreferrer' : undefined}
+          role={url === undefined ? 'menuitem' : undefined}
```

Comment:
same applies here...since parent container can either be menu or listbox, child list should have role menuitem or option.